### ISSUE TITLE
AdvancedMarker draggable doesn't work

### ISSUE THREAD

**Issue by osmanonurcan** [visgl/react-google-maps#50]
Using Next.js 13.5.6,
@vis.gl/react-google-maps: 0.1.2

```ruby
<APIProvider apiKey={API_KEY} libraries={['marker']}>
      <Map
        mapId={'DEMO_MAP_ID'}
        zoom={10}
        center={{lat: 0, lng: 0}}
        gestureHandling={'greedy'}
        disableDefaultUI={true}>
        <AdvancedMarker
          position={{lat: 20, lng: 10}}
          draggable={true}>
        </AdvancedMarker>
      </Map>
</APIProvider>
```

**Comment by usefulthink** [comment]
I just remembered that a while ago the maps-API changed the name of that option from `draggable` to `gmpDraggable` to avoid a conflict with the html-attribute with the same name. 

That might be something that causes problems here (I think I still want to keep the `draggable` naming, but for now it might be worth checking. Otherwise we can look into this in the coming days.

**Comment by usefulthink** [comment]
Your fix is now published in v0.2.0.
